### ISSUE TITLE
🐞 `dagger -i` that should return a service, does not work

### ISSUE THREAD

**Issue by jpadams** [dagger/dagger#8718]
### What is the issue?

🐞 `dagger -i` that should return a service, does not work

### Dagger version

dagger v0.15.5

### Steps to reproduce

🐞 `dagger -i` that should return a service, does not work

### Log output

🐞 `dagger -i` that should return a service, does not work

**Comment by jedevc** [comment]
@jpadams do you remember what this one is about?

I can't remember what this is, or what the expected behavior should be.

**Comment by jpadams** [comment]
> [@jpadams](https://github.com/jpadams) do you remember what this one is about?
> 
> I can't remember what this is, or what the expected behavior should be.

@jedevc 
Perhaps the feeling of inconsistency of return type of `container` being seemingly "synced" thus triggering interactive debugging terminal while return type of `service` is not for the same erroring `container`.  I guess we just wait for the `up` to trigger sync of the `container` for `service` return type.

```
# triggers interactive debug terminal
dagger -i core container from --address nginx with-exec --args exit,1  with-exposed-port --port 80
```
```
# DOES NOT trigger interactive debug terminal
dagger -i core container from --address nginx with-exec --args exit,1  with-exposed-port --port 80 as-service
```

```
# triggers interactive debug terminal
dagger -i core container from --address nginx with-exec --args exit,1  with-exposed-port --port 80 as-service up
```

But...that is prob not the original problem...

I think 0.15.5 was just a cursed version and that we struck it from the record. I can't download it.

![Image](https://github.com/user-attachments/assets/8870ee44-31c2-4b8a-9236-733c6d8d0030)

**Comment by jpadams** [comment]
Yep, let's assume it was a transient 0.15.5 problem that was fixed in 0.16.x
